### DISCUSSION TITLE
CPP 20 compatibility

### DISCUSSION THREAD

**Discussion by zlnimda** [nlohmann/json#4106] [Ideas]
Hello there,

I recently transitioned a code base to cpp20, and `<ciso646>` has been deprecated.
It is recommended to use `<version>` instead.

This does not cause any major problem currently on msbuild with vs2022, but might not be compatible later.

**Comment by nlohmann** [comment]
Good news: this appears to already be resolved. The current header no longer includes or references `<ciso646>` anywhere in the codebase, so the C++20 deprecation warning this discussion raised should no longer occur on current releases.

---
*This reply was drafted by [Claude Code](https://claude.com/claude-code) on behalf of @nlohmann, as part of a review of open discussions.*
